src/main/java/com/elmakers/mine/bukkit/action/builtin/CustomProjectileAction.java

```diff
@@ -649,8 +710,17 @@ else if (reorient)
         if (!block.getChunk().isLoaded()) {
             return miss();
         }
-
-        if (targetingResult == Targeting.TargetingResult.BLOCK) {
+        
+        if(distanceTravelled<min_range && targetingResult != null) {
+        	if(distanceTravelled>=min_block_range && targetingResult == Targeting.TargetingResult.BLOCK) {
```

Comment:
Or, maybe rather if it is meant to be >=, shouldn't these return hit() ?

Maybe I will look at this again in the morning after some sleep :D